configuration/restdocs.md

```diff
@@ -7,14 +7,45 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
+Through the openHAB [REST API](https://en.wikipedia.org/wiki/REST_API) most aspects of the openHAB system can be accessed by other programs.  
+This includes for example, the access to all data related to Items, Things and Bindings as well as the capabilities to invoke  actions that change the state of Items. 
+Interactions with the REST API are based on the http protocol and therefore can exchange among different computer systems. 
+The information available from the API includes Items and their state, installed bindings, sitemap information, and much more. 
+The REST API provides an interface that can only be accessed programmatically.
+However, from a technical perspective it is very easy to use. 
+Be aware that the REST API is not automatically installed in openHAB. 
+After installation, the REST APT is accessible through the openHAB dashboard.
+Access over the Internet to the REST API is possible, but this represents a significant security risk. 
+Users are encouraged to [ensure safe and secure connections](http://docs.openhab.org/installation/security.html). 
 
-For further details, please check:
+## REST API Examples
 
-- The interactive REST API documentation available through the openHAB 2 dashboard (Add-on "misc-restdocs", installed by default)
-- [The openHAB 1.x wiki REST API article](https://github.com/openhab/openhab/wiki/REST-API)
+- Using third party software on cell phones, such as tasker to [open your garage door](https://community.openhab.org/t/triggering-items-using-openhab-2s-rest-api-from-tasker/14027)
+- Interacting with openHAB via other programs; many computer languages can easily make use of REST APIs
+- Accessing of information from devices that support http communication
+- Updating Item states through external devices that are capable of sending http commands (for example, some motion detectors or surveillance cameras)
```

Comment:
I'd start with the generic examples first and move the cell phone example to the bottom.

As for the generic ones, how about something like:

*    Retrieve openHAB data from external applications
*    Inject data and trigger events in openHAB from external applications
*    Inspect openHAB Bindings/Things or Items, learn about current states, parameters or problems